schemas/sdk.schema

```diff
@@ -727,16 +727,11 @@
                 },
                 "allowInterruptions": {
                     "type": "string",
-                    "enum": [
-                        "always",
-                        "never",
-                        "notRecognized"
-                    ],
                     "title": "Allow Interruptions",
-                    "description": "Always will always consult parent dialogs first, never will not consult parent dialogs, notRecognized will consult parent only when it's not recognized",
-                    "default": "notRecognized",
+                    "description": "A boolean expression that determines whether the parent should be allowed to interrupt the input.",
+                    "default": "true",
                     "examples": [
-                        "notRecognized"
+                        "true"
```

Comment:
You shouldn't edit theses by hand, there is a Microsoft.TextInput.schema which we run through schema merge tool to generate the merged view of all of the schemas.